src/stores/SettingsStore.ts

```diff
@@ -25,12 +26,13 @@ const defaultValues: SettingsType = {
   affectsPerPage: 10,
   trackersPerPage: 10,
   isHidingLabels: false,
+  privacyNoticeShown: false,
 };
-const osimSettings = useStorage('OSIM::USER-SETTINGS', structuredClone(defaultValues));
 
 export const useSettingsStore = defineStore('SettingsStore', () => {
-  const settings = ref<SettingsType>(osimSettings.value);
-  // const settings = useSessionStorage(_settingsStoreKey, {} as SettingsType);
+  const { addToast } = useToastStore();
+
+  const settings = useLocalStorage('OSIM::USER-SETTINGS', structuredClone(defaultValues));
```

Comment:
>Should we use a pinia plugin for persisting data? 
We are handling this differently on each store, maybe is worth centralizing the logic to one place.
We could write our own plugin or use the community one https://github.com/prazdevs/pinia-plugin-persistedstate